lib/placeholders/box/readMe.md

```diff
@@ -0,0 +1,31 @@
+## Navbar
+
+The box placeholder component.
+
+### Properties
+
+| Name     | Type     | Default   | Description                          |
+| :------- | :------- | :-------- | :----------------------------------- |
+| `square` | `circle` | &nbsp;    | Shapes of the box default is circle. |
```

Comment:
I can't see `square` prop used anywhere in the component. Is this by mistake?

Reply:
07e1bd894e713bbe71d96aaf554badda90d852db